Makefile

```diff
@@ -50,13 +50,13 @@ format-cfn:
 
 .PHONY: format-docs
 format-docs:
-	npx prettier-eslint ./*.md ./docs/*.md --write --prose-wrap always
+	npx prettier ./*.md ./docs/*.md --write
```

Comment:
Create React App v5 moved internally to ESLint 8, and prettier-eslint cli is stuck to ESLint 5: https://github.com/prettier/prettier-eslint-cli/issues/304

When upgrading, prettier-eslint was failing. I decided to just move to prettier cli, which has a `--write` functionality as well. The difference is that instead of inferring the linting config from the Create React App module, I am defining the config inside the root's `package.json`. But the advantage is that the dependency tree is much smaller and that allows to remove some vulnerabilities too.

When running with just prettier, the linting is 99% same. You'll notice a couple of new line changes in a couple of files, but I decided that was ok for now.